dashboards/f5/f5-big-ip.json

```diff
@@ -0,0 +1,1184 @@
+                }
+              ],
+              "platformOptions": {
+                "ignoreTimeRange": false
+              },
+              "units": {
+                "unit": "BITS_PER_SECOND"
+              },
+              "yAxisLeft": {
+                "zero": true
+              }
+            }
+          },
+          {
+            "title": "Top 10 Transmit bps",
+            "layout": {
+              "column": 7,
+              "row": 9,
+              "width": 6,
+              "height": 4
+            },
+            "linkedEntityGuids": null,
+            "visualization": {
+              "id": "viz.line"
+            },
+            "rawConfiguration": {
+              "facet": {
+                "showOtherSeries": false
+              },
+              "legend": {
+                "enabled": true
+              },
+              "nrqlQueries": [
+                {
+                  "accountIds": [],
+                  "query": "FROM F5BigIpNodeSample SELECT average(node.outDataInBytesPerSecond) AS 'Tx bps' WHERE reportingEndpoint IN ( {{f5_system}} ) AND displayName IN ( {{f5_node}} ) FACET reportingEndpoint AS 'System', displayName AS 'Node' TIMESERIES LIMIT 10"
+                }
+              ],
+              "platformOptions": {
+                "ignoreTimeRange": false
+              },
+              "units": {
+                "unit": "BITS_PER_SECOND"
+              },
+              "yAxisLeft": {
+                "zero": true
+              }
+            }
+          }
+        ]
+      },
+      {
+        "name": "Pools",
+        "description": null,
+        "widgets": [
+          {
+            "title": "",
+            "layout": {
+              "column": 1,
+              "row": 1,
+              "width": 3,
+              "height": 4
+            },
+            "linkedEntityGuids": null,
+            "visualization": {
+              "id": "viz.markdown"
+            },
+            "rawConfiguration": {
+              "text": "![f5 logo](https://sjc04pap001files.storage.live.com/y4mmTKKQgYW0voEkAP7b-jMes8Eoopinjpvx1iMaO0omXrOF6mG9nlRWai-CilsMmC6UT_KTGYpCNmIlMqHBYIx5VQgSWKSCcF0YVcw8zH8iRsYWNkqmu-rljXIVkaG-2WooObwwbKZmYCILlEMIMqxRZLtvp2UMOjgpG-DlwXcuYR3IodghiBh-br0MMMgB0qvnkYJK6weLCzvRJc5i4jVoQ?encodeFailures=1&width=110&height=110)\n\n### Pools\nA [pool](https://techdocs.f5.com/kb/en-us/products/big-ip_ltm/manuals/product/ltm-concepts-11-5-1/6.html#conceptid) is a logical set of devices, such as web servers, that you group together to receive and process traffic. Instead of sending client traffic to the destination IP address specified in the client request, Local Traffic Manager™ sends the request to any of the servers that are members of that pool. This helps to efficiently distribute the load on your server resources."
```

Comment:
Lets update this one as well:
https://newrelic.com/sites/default/files/quickstarts/images/icons/f5--logo.svg